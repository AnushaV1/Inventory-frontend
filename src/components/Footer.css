.footer {
    position: fixed;
    padding-top:20px;
    margin-top: 50px;
    bottom: 0;
    width: 100%;
    height: 60px;
    line-height: 60px; 
    background-color: #1d1a1a;
    
}

