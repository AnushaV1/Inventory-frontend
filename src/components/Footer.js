import React from 'react';
import './Footer.css'

const Footer = () => {
	return (
		<footer className="footer">
		<div className="container-fluid text-center">
        <div className="row">
			<div className="col-md-4 mt-md-0 mt-3 text-left">
			<h5><a href="https://www.upcitemdb.com/api/explorer#!/lookup/get_trial_lookup" className="text-white-50">API - UPCItemDB </a></h5>
            </div>
			<div className="col-md-4 mb-md-0 mb-3 ">
			<h5 className="text-white-50">Springboard Capstone project 2</h5>
		</div>

            <div className="col-md-4 mb-md-0 mb-3 text-center">
            <h4> <a href="http://www.linkedin.com/in/anusha-venkataraghavan" className="text-white-50"><i className="fab fa-linkedin"></i></a></h4>
			</div>

		</div>
        </div>
    </footer>
	);
};

export default Footer;

