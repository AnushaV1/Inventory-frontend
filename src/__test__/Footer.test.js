import React from 'react';
import { render } from '@testing-library/react';
import Footer from '../components/Footer';


it('renders without crashing', function () {
	render(<Footer />);
})

it('matches snapshot', function () {
	const { asFragment } = render(<Footer />);
	expect(asFragment()).toMatchSnapshot();
});