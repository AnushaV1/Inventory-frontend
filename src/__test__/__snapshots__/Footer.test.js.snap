// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <div
      class="container-fluid text-center"
    >
      <div
        class="row"
      >
        <div
          class="col-md-4 mt-md-0 mt-3 text-left"
        >
          <h5>
            <a
              class="text-white-50"
              href="https://www.upcitemdb.com/api/explorer#!/lookup/get_trial_lookup"
            >
              API - UPCItemDB 
            </a>
          </h5>
        </div>
        <div
          class="col-md-4 mb-md-0 mb-3 "
        >
          <h5
            class="text-white-50"
          >
            Springboard Capstone project 2
          </h5>
        </div>
        <div
          class="col-md-4 mb-md-0 mb-3 text-center"
        >
          <h4>
             
            <a
              class="text-white-50"
              href="http://www.linkedin.com/in/anusha-venkataraghavan"
            >
              <i
                class="fab fa-linkedin"
              />
            </a>
          </h4>
        </div>
      </div>
    </div>
  </footer>
</DocumentFragment>
`;
